package.json

```diff
@@ -6,6 +6,7 @@
   "nativePackage": true,
   "dependencies": {
     "npmlog": "^2.0.4",
+    "promise": "^7.1.1",
```

Comment:
I don't think we should pin to more than the major version here, since we have a very small dependency and promise is likely to show up elsewhere in the dependency tree.